src/main/java/com/google/cloud/genomics/dataflow/readers/VariantStreamer.java

```diff
@@ -79,7 +81,7 @@ public static StreamVariantsRequest getVariantRequests(String variantSetId) {
         .transformAndConcat(new Function<Contig, Iterable<Contig>>() {
             @Override
             public Iterable<Contig> apply(Contig contig) {
-              return contig.getShards(100000000L);
+              return contig.getShards(5000000L);
```

Comment:
See my previous comment regarding hardcoding it:

https://github.com/googlegenomics/dataflow-java/pull/106/files#r33701593